pkg/packagekit/updates.jsx

```diff
@@ -672,18 +672,27 @@ const TwoColumnTitle = ({ icon, str }) => {
 
 const UpdateSuccess = ({ onIgnore, openServiceRestartDialog, openRebootDialog, restart, manual, reboot, tracerAvailable, history }) => {
     if (!tracerAvailable) {
+        /* tracer is not available any more in RHEL 10; as a special case, if only kpatch and kernel were
```

Comment:
This feels like a knee-jerk reaction triggered by an unexpected change in rhel-10, which I guess is ok for rhel-10. But it affects all OSes without tracer like Debian and Ubuntu. Is that okay?

Reply:
Conceptually, I think yes. Practically there's no difference -- kpatch is only  a thing on RHEL and Ubuntu with a commercial Canonical "Ubuntu Advantage" plan or so.

Reply:
Ah, I see.